Library/Homebrew/os/mac/mach.rb

```diff
@@ -202,3 +199,7 @@ def dylib_id
     mach_metadata.dylib_id
   end
 end
+
+class Pathname
+  include(MachO)
+end
```

Comment:
Yeh, I prefer all monkeypatching happens in `Library/Homebrew/extend/` somewhere.